lib/openid_connect.ex

```diff
@@ -481,13 +489,19 @@ defmodule OpenIDConnect do
 
   1. The token's signature is valid and was signed by the provider's key
   2. The token has not expired (checking the "exp" claim)
-  3. The token is intended for your application (checking the "aud" claim)
+  3. The token is intended for your application (checking the "aud" claim). This validation can
+     be skipped by using the `ignore_audience_matching?: true` option.
 
   ## Parameters
 
   * `config` - The provider configuration map
   * `jwt` - The ID token string (a JSON Web Token) from the tokens response
 
+  ## Options
+
+  - `:ignore_audience_matching?` - If set to `true`, the `"aud"` claim value will not be
```

Comment:
I'm not sure about this as an explicit option. It may be better and more flexible if you could pass a kw list of any options to be ignored rather than one offs

Reply:
yeah, that way it can adapt to any customization

Reply:
- Adjusted: https://github.com/DockYard/openid_connect/commit/36bc0638e4789b94da388e841a62d395c41dddcc